index.js

```diff
@@ -18,11 +20,12 @@ const {SignatureSigner} = require('./signer');
  */
 class ClientServiceBase {
   /**
-   * @param {LedgerClient} ledgerClient LedgerClient object to inject
+   * @param {Object} service contains the object of ledgeClient and
+   *  the object of ledgerPrivileged
    * @param {Protobuf} protobuf protobuf object to inject
    * @param {Object} properties JSON Object used for setting client properties
    */
-  constructor(ledgerClient, protobuf, properties) {
+  constructor(service, protobuf, properties) {
```

Comment:
I'm not sure about the convention of  javascript, but I feel `service` is not a single entry so `services` sounds better to me.